README.md

```diff
@@ -17,6 +17,8 @@ FHIR Works on AWS utilizes AWS Lambda, Amazon DynamoDB, Amazon S3 and Amazon Ela
 
 The easiest and quickest way to access FHIR Works on AWS is by using [AWS solution](https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/). To modify the code and set up a developer environment, follow the steps below:
 
+**Note**: AWS Solution provides an earlier version of FWoA install, please follow the instruction below to install from GitHub repository if you wish to install the latest version.
```

Comment:
Can you add a link to the AWS solution here?

I think that we don't mention the solution anywhere else and without a link most customers won't understand this note

Reply:
Sure, I'll add a link to our solutions branch

Reply:
I was thinking of a link to the Soultion page: https://aws.amazon.com/solutions/implementations/fhir-works-on-aws/?did=sl_card&trk=sl_card

Reply:
The Solutions link is present on line 18